confignode/src/main/java/org/apache/iotdb/confignode/persistence/ClusterSchemaInfo.java

```diff
@@ -43,23 +43,33 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
 import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
+import java.util.UUID;
 import java.util.concurrent.locks.ReentrantReadWriteLock;
 
-public class ClusterSchemaInfo {
+public class ClusterSchemaInfo implements SnapshotProcessor {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ClusterSchemaInfo.class);
 
   // StorageGroup read write lock
   private final ReentrantReadWriteLock storageGroupReadWriteLock;
 
-  // TODO: serialize and deserialize
   private MTreeAboveSG mTree;
 
+  // The size of the buffer used for snapshot(temporary value)
+  private final int bufferSize = 10 * 1024 * 1024;
+
+  private final String snapshotFileName = "ClusterSchemaInfo.st";
```

Comment:
ClusterSchemaInfo is much like a class name and looks like IoTDB names file by camel naming pattern. How about use "cluster_schema.bin" as the file name.

Reply:
+1

Reply:
fixed